modules/tinymce/src/core/test/ts/browser/dom/ScrollIntoViewTest.ts

```diff
@@ -241,3 +258,48 @@ describe('browser.tinymce.core.dom.ScrollIntoViewTest', () => {
     });
   });
 });
+
+context('scrollToMarker should not scroll the container', () => {
+  const insertNewline = (editor: Editor, args: Partial<EditorEvent<KeyboardEvent>>) => {
+    InsertNewLine.insert(editor, args as EditorEvent<KeyboardEvent>);
+  };
+  const container: SugarElement<HTMLButtonElement> = SugarElement.fromHtml('<div id="ephox-ui" style="border: 2px solid red; margin-top: 4000px; margin-bottom: 4000px"></div>');
```

Comment:
why the `HTMLButtonElement` type is used here?

Reply:
I forgot to change after a copy paste, now I fixed it